k8s/nginx/production/global-config.yaml

```diff
@@ -34,8 +34,10 @@ data:
       if ($request_uri ~ '/airflow/api/v') { return 200; }    
       if ($request_uri ~ '/argocd') { return 200; }    
       if ($request_uri !~ '/api/v') { return 200; } # None API requests 
+      if ($request_uri ~ '/api/v2/users/updatePasswordViaEmail') { return 200; }
       
       if ($request_uri ~ '/api/v1') { return 200; } # API v1 endpoint requests
+      if ($request_method = 'POST' && $request_uri ~ '${BASE_AUTH_URL}/users') { return 200; }
```

Comment:
Hi @Kisakye-e 
Please update this to the actual value since we do not have` BASE_AUTH_URL` here. Let @Codebmk share the actual path accordingly. Thanks!

Reply:
@Baalmart , waiting for the actual path from @Codebmk

Reply:
@Kisakye-e these are the api routes
`/api/v2/users`
`/api/v2/users/updatePasswordViaEmail`

Reply:
Thank you @Codebmk .

@Baalmart , this has been sorted

Reply:
Hi @Kisakye-e , the `/api/v2/users/updatePasswordViaEmail` is a PUT method, have we taken that into consideration?
cc: @Codebmk